capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_integration_test.go

```diff
@@ -71,7 +71,157 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+func Test_removeNoLongerExistingVersionsInfos(t *testing.T) {
+	var ctx = context.Background()
+	var endpointURL = "https://example.com"
+	// add vendor that can later be referenced
+	fakeVendor := &endpointmanager.Vendor{
+		Name:          "Fake Vendor",
+		DeveloperCode: "12345",
+		CHPLID:        222,
+	}
+	var endpointMetadataDefaultRequest = &endpointmanager.FHIREndpointMetadata{
+		URL:               endpointURL,
+		HTTPResponse:      200,
+		Errors:            "",
+		SMARTHTTPResponse: 0,
+		Availability:      1.0}
+
+	var endpointMetadataRequestVersion1 = &endpointmanager.FHIREndpointMetadata{
+		URL:               endpointURL,
+		HTTPResponse:      200,
+		Errors:            "",
+		SMARTHTTPResponse: 0,
+		Availability:      1.0}
+
+	// endpointInfos
+	var endpointInfoDefaultRequest = &endpointmanager.FHIREndpointInfo{
+		URL:                   "https://example.com",
+		VendorID:              fakeVendor.ID,
+		TLSVersion:            "TLS 1.1",
+		MIMETypes:             []string{"application/json+fhir"},
+		CapabilityStatement:   nil,
+		SMARTResponse:         nil,
+		RequestedFhirVersion:  "None",
+		CapabilityFhirVersion: "1.0.2",
+		Metadata:              endpointMetadataDefaultRequest}
+
+	var endpointInfo1RequestedVersion1 = &endpointmanager.FHIREndpointInfo{
+		URL:                   "https://example.com",
+		VendorID:              fakeVendor.ID,
+		TLSVersion:            "TLS 1.1",
+		MIMETypes:             []string{"application/json+fhir"},
+		CapabilityStatement:   nil,
+		SMARTResponse:         nil,
+		RequestedFhirVersion:  "1.0.0",
+		CapabilityFhirVersion: "1.0.2",
+		Metadata:              endpointMetadataRequestVersion1}
+
+	err := setup()
```

Comment:
Why is `setup()` called here and in `Test_saveMsgInDB` if it's also called in the `TestMain` function?

Reply:
I think this was an artifact of my debugging. I removed it and it doesn't appear to have changed the test behavior.